src/accordion/accordion-item.component.ts

```diff
@@ -21,17 +21,29 @@ import {
 				viewBox="0 0 7 12">
           		<path fill-rule="nonzero" d="M5.569 5.994L0 .726.687 0l6.336 5.994-6.335 6.002L0 11.27z"/>
 			</svg>
-			 <p class="bx--accordion__title">{{title}}</p>
+			<p
+				class="bx--accordion__title"
+				[ngClass]="{
+					'bx--skeleton__text': skeleton
+				}">
+				{{!skeleton ? title : null}}
```

Comment:
Is it bad if we show the title regardless if it's available?

Reply:
If its available it will show in the loading state.. which we don't want happening